src/org/labkey/test/tests/SampleSetParentColumnTest.java

```diff
@@ -691,7 +692,7 @@ public void testAliasNameConflictsWithFieldName()
         log("Add a parent alias column to the sample set that conflicts with a given column name.");
         sampleHelper.goToEditSampleSet(SAMPLE_SET_NAME);
 
-        sampleHelper.addParentColumnAlias(Map.of(ALIAS_NAME_CONFLICT, SAMPLE_SET_NAME));
+        sampleHelper.addParentColumnAlias(Map.of(ALIAS_NAME_CONFLICT, CURRENT_SAMPLE_SET_OPTION));
```

Comment:
This works but it needs to be done for all the tests that have failed:
testParentInSameSampleSet
testValidAliasNames
testMaterialInputsWorkWithAliases
testUseThenRemoveAnAliasParentColumn
testMultipleParentColumns